### ISSUE TITLE
Improve humanization of dates on the website

### ISSUE THREAD

**Issue by augustoproiete** [cake-build/website#1476]
The humanization of dates is resolved during build time therefore it gets out of sequence.

![image](https://user-images.githubusercontent.com/177608/104478981-491e1e80-5599-11eb-8045-7cae70d67727.png)

Some possible alternatives to consider:

- Do the humanization of dates on the client-side instead
- Set up a more frequent deployment (e.g. nightly)
    - doesn't completely resolve the issue, but improves it
- (something else?)

**Comment by daveaglick** [comment]
Client-side is probably best - that way we can even customize it for the browser culture if we want to

**Comment by devlead** [comment]
> Client-side is probably best - that way we can even customize it for the browser culture if we want to

Agreed and if we have something like a Unix timestamp as the data it's easy to get right in js and will respect the viewer's timezone.

**Comment by fierc3** [comment]
I'd like to help out with this issue

**Comment by TheRealRajdeep** [comment]
Hey, I saw that no one has been working on the issue and decided to solve it. Here are the results. Feel free to have a look at them. If you allow, I can raise a PR then.

<img width="317" height="205" alt="Image" src="https://github.com/user-attachments/assets/e2585fec-e590-4dfd-b19d-6f476bdbbb54" />

<img width="307" height="124" alt="Image" src="https://github.com/user-attachments/assets/054057f4-9cc5-418a-b9b9-829308768781" />

**Comment by poonambhagaur61-tech** [comment]
I am thinking of implementing client-side date humanization using JavaScript so that the time updates dynamically instead of being fixed at build time.
Please let me know if this approach works.

**Comment by Bahi0** [comment]
Hi! I’m new to open-source contributions and interested in working on this issue if it’s still available.

I noticed the earlier suggestion about implementing client-side date humanization using JavaScript. I’d like to try that approach so the dates update dynamically after deployment instead of staying fixed at build time.

Please let me know if this still aligns with the current direction.

**Comment by Bahi0** [comment]
Hi! I’m new to open-source contributions and would like to work on this issue if it’s still available.

I noticed the earlier suggestion about implementing client-side date humanization using JavaScript. I’d like to try that approach so the dates update dynamically after deployment instead of remaining fixed at build time.

Please let me know if this still aligns with the current direction.
